db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/SchemaDescriptor.java

```diff
@@ -97,7 +97,8 @@
     public static final	String	IBM_SYSTEM_STAT_SCHEMA_NAME     = "SYSSTAT";
     public static final	String	IBM_SYSTEM_NULLID_SCHEMA_NAME   = "NULLID";
 
-    /**
+
+	/**
```

Comment:
nit: remove the additional leading spaces

Reply:
Fixed